Extension/package.nls.json

```diff
@@ -24,25 +24,25 @@
     "c_cpp.configuration.formatting.Disabled.description": "Code formatting will be disabled",
     "c_cpp.configuration.vcFormat.indent.braces.description": "Braces are indented by the amount specified in the Editor: Tab Size setting",
     "c_cpp.configuration.vcFormat.indent.multiLineRelativeTo.description": "Indent new lines relatively to:",
-    "c_cpp.configuration.vcFormat.indent.multiLineRelativeTo.outermostParenthesis.description": "the outermost open parenthesis",
-    "c_cpp.configuration.vcFormat.indent.multiLineRelativeTo.innermostParenthesis.description": "the innermost open parenthesis",
-    "c_cpp.configuration.vcFormat.indent.multiLineRelativeTo.statementBegin.description": "the beginning of the current statement",
+    "c_cpp.configuration.vcFormat.indent.multiLineRelativeTo.outermostParenthesis.description": "The outermost open parenthesis",
+    "c_cpp.configuration.vcFormat.indent.multiLineRelativeTo.innermostParenthesis.description": "The innermost open parenthesis",
+    "c_cpp.configuration.vcFormat.indent.multiLineRelativeTo.statementBegin.description": "The beginning of the current statement",
     "c_cpp.configuration.vcFormat.indent.withinParentheses.description": "When a new line is typed, it is aligned under the opening parenthesis or based on `C_Cpp.vcFormat.indent.multiLineRelativeTo`",
     "c_cpp.configuration.vcFormat.indent.withinParentheses.alignToParenthesis.description": "New line is aligned under the opening parenthesis",
     "c_cpp.configuration.vcFormat.indent.withinParentheses.indent.description": "New line is indented based on `C_Cpp.vcFormat.indent.multiLineRelativeTo`",
     "c_cpp.configuration.vcFormat.indent.preserveWithinParentheses.description": "In existing code, preserve the existing indent alignment of new lines within parentheses",
     "c_cpp.configuration.vcFormat.indent.caseLabels.description": "Labels are indented relative to switch statements by the amount specified in the Editor: Tab Size setting",
-    "c_cpp.configuration.vcFormat.indent.caseContents.description": "Code inside case is indented relative to its label by the amount specified in the Editor: Tab Size setting",
+    "c_cpp.configuration.vcFormat.indent.caseContents.description": "Code inside case block is indented relative to its label by the amount specified in the Editor: Tab Size setting",
```

Comment:
Keep this change, but I think the capitalizations should be reverted.

Reply:
Maybe they should be changed to be complete sentences? Do you know of another example of an enum value not having a complete sentence description?

Reply:
I don't think it makes sense for these to be non-capitalized partial sentences.  This does not consider other languages in which it may not be possible to split these sentences in the same way.  It's also difficult to provide guidance to the Loc team on how to maintain these strings if not self-contained (as there isn't a way to add hints to package.nls.json).

Reply:
The only prior use of `enumDescriptions` was `C_Cpp.formatting`, which are complete sentences, were not continuations of a sentence, and were capitalized (except when starting with a 'clang-format', as the binary name is not capitalized).  Note that these enum descriptions are also not continuations of a sentence.  So, I made them complete sentences.  Also, I omitted periods, as we generally do not appear to be using them in this file, even for complete sentences.

Reply:
That seems good with the sentences and capitalization, but I'm seeing the VS Code setting descriptions end with periods? Can we update them all to end in periods?